### DISCUSSION TITLE
Disable breadcrumb for specific resources

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#9997] [Help]
How can I disable breadcrumbs for specific resources?

**Comment by leandrocfe** [comment]
You can use this in the Resource

```php
public static function getBreadcrumb(): string
{
    return '';
}
```

this in the Pages

```php
public function getBreadcrumb(): ?string
{
    return null;
}
```

**Reply by ArtMin96** [reply]
Already got that. Thanks @leandrocfe

**Reply by alisalehi1380** [reply]
Thanks @leandrocfe ✅

**Reply by indigoram89** [reply]
@leandrocfe Hi! This is does not work anymore? I have a Cluster with Resource:

<img width="446" height="362" alt="CleanShot 2026-05-20 at 11 13 30@2x" src="https://github.com/user-attachments/assets/c9eba9c2-2bff-4868-b12d-3a2f013be277" />

Thank you!
